share/man/man3/sysexits.3

```diff
@@ -39,9 +39,10 @@ it is not a good practice to call
 .Xr exit 3
 with arbitrary values to indicate a failure condition when ending
 a program.
-Instead, the pre-defined exit codes from
-.Nm
-should be used, so the caller of the process can get a rough
+Instead, they should exit 0 on success and 1 on failure.
+Historically, an attempt was made to standardize pre-defined exit codes in
+.Nm ,
+so the caller of the process can get a rough
 estimation about the failure class without looking up the source code.
```

Comment:
I'd replace this entire paragraph with:
This interface has been deprecated and exists only for compatibility with 3rd party software. Its use is discouraged.

Reply:
Some programs that still use them are not third party though. How's the latest one?